api/src/organizational-entities/domain/errors.js

```diff
@@ -32,6 +40,14 @@ class OrganizationNotFound extends DomainError {
   }
 }
 
+class OrganizationBatchUpdateError extends DomainError {
+  constructor({ code = 'ORGANIZATION_BATCH_UPDATE_ERROR', message = 'Organization batch update failed', meta } = {}) {
```

Comment:
Rien d'obligatoire, mais je pense qu'on gagne à ce que cette erreur ait le qualificatif « generic » quelque part dans son nom, en effet les autres erreurs sont aussi des OrganizationBatchUpdateErrors, mais elles elles sont spécifiques.

```suggestion
class GenericOrganizationBatchUpdateError extends DomainError {
  constructor({ code = 'GENERIC_ORGANIZATION_BATCH_UPDATE_ERROR', message = 'Generic organization batch update failed', meta } = {}) {
```